subsys/trusted_storage/src/settings_helpers.c

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/kernel.h>
+#include <zephyr/init.h>
+#include <zephyr/settings/settings.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/logging/log.h>
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stddef.h>
+#include <string.h>
+#include <errno.h>
+
+#include "settings_helpers.h"
+
+/* Helper to fill filename with a suffix */
+#define TRUSTED_STORAGE_FILENAME_FILL(filename, prefix, uid, suffix)                               \
+	snprintf((filename), TRUSTED_STORAGE_FILENAME_MAX_LENGTH + 1,                              \
```

Comment:
snprintf return value not checked (not obvious to me if it's safe to skip)

Reply:
Added it at places where this is used

Reply:
Added the extra check